doc/en/example/simple.rst

```diff
@@ -699,36 +699,33 @@ and run it::
 You'll see that the fixture finalizers could use the precise reporting
 information.
 
-Integrating pytest runner and cx_freeze
------------------------------------------------------------
+Integrating pytest runner and PyInstaller
+-----------------------------------------
 
 If you freeze your application using a tool like
-`cx_freeze <https://cx-freeze.readthedocs.io>`_ in order to distribute it
-to your end-users, it is a good idea to also package your test runner and run
-your tests using the frozen application.
-
-This way packaging errors such as dependencies not being
-included into the executable can be detected early while also allowing you to
-send test files to users so they can run them in their machines, which can be
-invaluable to obtain more information about a hard to reproduce bug.
-
-Unfortunately ``cx_freeze`` can't discover them
+`PyInstaller <https://pyinstaller.readthedocs.io>`_
+in order to distribute it to your end-users, it is a good idea to also package
+your test runner and run your tests using the frozen application. This way packaging
+errors such as dependencies not being included into the executable can be detected early
+while also allowing you to send test files to users so they can run them in their
+machines, which can be invaluable to obtain more information about a hard to reproduce bug.
+
+Unfortunately ``PyInstaller`` can't discover them
 automatically because of ``pytest``'s use of dynamic module loading, so you
-must declare them explicitly by using ``pytest.freeze_includes()``::
+must declare them explicitly by using ``pytest.freeze_includes()`` and an
+auxiliary script:
 
-    # contents of setup.py
-    from cx_Freeze import setup, Executable
+.. code-block:: python
+
+    # contents of create_executable.py
     import pytest
+    import subprocess
 
-    setup(
-        name="app_main",
-        executables=[Executable("app_main.py")],
-        options={"build_exe":
-            {
-            'includes': pytest.freeze_includes()}
-            },
-        # ... other options
-    )
+    hidden = []
+    for x in pytest.freeze_includes():
+        hidden.extend(['--hidden-import', x])
+    args = ['pyinstaller'] + hidden + ['runtests_script.py']
+    subprocess.check_call(' '.join(args), shell=True)
```

Comment:
> Why not simply subprocess.check_call(args) without shell=True? 

On Windows without `shell=True` you have to add the `.exe` extension:

```
>>> import subprocess
>>> subprocess.check_call('py.test --help')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "e:\ws\Souring\envs\steam_souring10\lib\subprocess.py", line 536, in check_call
    retcode = call(*popenargs, **kwargs)
  File "e:\ws\Souring\envs\steam_souring10\lib\subprocess.py", line 523, in call
    return Popen(*popenargs, **kwargs).wait()
  File "e:\ws\Souring\envs\steam_souring10\lib\subprocess.py", line 711, in __init__
    errread, errwrite)
  File "e:\ws\Souring\envs\steam_souring10\lib\subprocess.py", line 959, in _execute_child
    startupinfo)
WindowsError: [Error 2] The system cannot find the file specified
>>> subprocess.check_call('py.test.exe --help', shell=True)
... <output> ...
0
```

And I wanted to keep things simple here. 

> Also, it might be better to tell users what to add to their spec file ...

My intention in this topic is not actually be a complete help on how to use PyInstaller, only to briefly demonstrate how to use the `pytest.freeze_includes()` function. Actually now that  pyinstaller/pyinstaller#2119 is merged, I think this docs should be updated to just mention that pytest should work out of the box with the latest PyInstaller, while others using other tools can use the `freeze_includes()` function to obtain the list of pytest internal modules. 

What do you think?